src/cmds/init.cmd.js

```diff
@@ -1,28 +1,32 @@
 import yargsInteractive from 'yargs-interactive'
 import initCommand from '../init';
-import { testsOptions } from './options-builder';
+import { testsOptions, credentialsOptions } from './options-builder';
 
 export const command = 'init';
 
 export const desc = 'init Perfecto and Cypress configuration files';
 
 export const builder = {
   prompt: { default: true, hidden: true, boolean: true },
-  'tests.path': {...testsOptions['tests.path']}
+  'tests.path': { ...testsOptions['tests.path']},
+  ...credentialsOptions,
+  skip: { alias: 's', type: 'boolean', default: false, describe: 'Skip interactive questions' },
```

Comment:
maybe the alias should be skip and not s? s it's too generic and  I think we use this already for security token

Reply:
I disabled the alias